docs/overview.rst

```diff
@@ -0,0 +1,213 @@
+============
+
+.. |logo-synchronisation| image:: images/logo-synchronisation.svg
+   :alt: https://thenounproject.com/search/?q=syncing&i=31170
+   :width: 100px
+
+.. |logo-offline| image:: images/logo-offline.svg
+   :alt: https://thenounproject.com/search/?q=offline&i=90580
+   :width: 100px
+
+.. |logo-permissions| image:: images/logo-permissions.svg
+   :alt: https://thenounproject.com/search/?q=permissions&i=23303
+   :width: 100px
+
+.. |logo-multiapps| image:: images/logo-multiapps.svg
+   :alt: https://thenounproject.com/search/?q=community&i=189189
+   :width: 100px
+
+.. |logo-selfhostable| image:: images/logo-selfhostable.svg
+   :alt: https://thenounproject.com/search/?q=free&i=669
+   :width: 100px
+
+.. |logo-community| image:: images/logo-community.svg
+   :alt: https://thenounproject.com/search/?q=community&i=189189
+   :width: 100px
+
++---------------------------------------------+-------------------------------------+
+| |logo-synchronisation|                      | |logo-offline|                      |
+| **Synchronisation and concurrency control** | **Offline-first JavaScript client** |
++---------------------------------------------+-------------------------------------+
+| |logo-permissions|                          | |logo-multiapps|                    |
+| **Fined grained permissions**               | **Plug multiple client apps**       |
++---------------------------------------------+-------------------------------------+
+| |logo-selfhostable|                         | |logo-community|                    |
+| **Open Source and Self-hostable**           | **Designed in the open**            |
++---------------------------------------------+-------------------------------------+
+
+
+**Also**
+
+- HTTP best practices
+- Pluggable authentication
+- Pluggable storage, cache, and permission backends
+- Configuration via a single INI file
+- Built-in monitoring
+
+
+**Coming soon**
+
+- Schema validation
+- Push notifications
+
+.. _overview-synchronisation:
+
+Synchronisation
+===============
+
+Bi-directionnal synchronisation of records is a very hard topic.
+
+*Kinto* takes some shortcuts by only providing the basics for concurrency control
+and polling for changes, and not trying to resolve conflicts automatically.
+
+Basically, each object has a revision number which is guaranteed to be incremented after
+each modification. *Kinto* does not keep any history.
+
+Clients can retrieve the list of changes that occured on a collection of records
+since a specified revision. *Kinto* can also use it to avoid accidental updates
+of objects.
+
+.. image:: images/overview-synchronisation.png
+    :align: center
+
+.. note::
+
+    *Kinto* synchronisation was designed and built by the `Mozilla Firefox Sync
+    <https://en.wikipedia.org/wiki/Firefox_Sync>`_ team.
+
+
+.. _comparison:
+
+Comparison with other solutions
+===============================
+
+Before starting to create yet another data storage service, we had a long
```

Comment:
Consider:
> Before we started on Yet Another Data Storage Service, we took a look at what was already out there, with a view to extending an existing community project (rather than reinventing the wheel). In the end, the solutions we reviewed didn't quite solve the problems we had - notably regarding fine-grained permission settings.
What follows is a comparison table showing how Kinto stacks up against some other projects in this space.